CMakeLists.txt

```
@@ -39,6 +40,12 @@ add_custom_target(luatest
   COMMENT "Run regression tests"
 )
 
+add_custom_target(luatest-no-coverage
```

It's better to do the opposite.
`luatest` -> `luatest-coverage`
`luatest-no-coverage` -> `luatest`
```suggestion
add_custom_target(luatest
```